app/cdap/components/WranglerV2/SearchResultsList/SearchResultsList.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+import SearchResultList from 'components/WranglerV2/SearchResultsList';
+
+export default {
+  title: 'SearchResultList',
+  component: SearchResultList,
+};
```

Comment:
refer to typescript example for storybooks and update the code with proper styles and code according to the official examples

Reply:
updated the story with typescript example